operations/configuration.md

```diff
@@ -420,6 +420,91 @@ Returns all tax environments supported by the API.
 | --- | --- | --- | --- |
 | `Value` | decimal | required | Tax rate, e.g. `0.21` in case of 21% tax rate. |
 
+## Get all accounting environments
+
+Returns all accounting environments supported by the API.
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/accountingEnvironments/getAll`
+
+```javascript
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+    "Client": "Sample Client 1.0.0"
+}
+```
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+
+### Response
+
+```javascript
+{
+    "AccountingEnvironments": [
+        {
+            "Code": "DEFAULT",
+            "EnglishName": "Default",
+            "CustomerLegalIdentifiers": [],
+            "CompanyLegalIdentifiers": []
+        },
+        {
+            "Code": "FR",
+            "EnglishName": "France",
+            "CustomerLegalIdentifiers": [],
+            "CompanyLegalIdentifiers": [
+                {
+                    "Code": "Siret"
+                }
+            ]
+        },
+        {
+            "Code": "IT",
+            "EnglishName": "Italy",
+            "CustomerLegalIdentifiers": [
+                {
+                    "Code": "ItDestinationCode"
+                },
+                {
+                    "Code": "ItFiscalCode"
+                }
+            ],
+            "CompanyLegalIdentifiers": [
+                {
+                    "Code": "ItDestinationCode"
+                },
+                {
+                    "Code": "ItFiscalCode"
+                }
+            ]
+        },
+}
+```
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `AccountingEnvironments` | array of [Accounting Environment](#accounting-environment) | required | The supported accounting environments. |
+
+#### Accounting Environment
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `Code` | string | required | Code of the environment. |
+| `EnglishName` | string | required | English name of the environment. |
+| `CustomerLegalIdentifiers` | string | array of [Legal Identifier](#legal-identifier) | Required identifiers for customers. |
```

Comment:
Actually, those are not general requirements on all customers. Those are only requirements for closing a bill, so I think this information should be somehow reflected here. CC @marektresnak  to clarify exact purpose of these.